lib/ethpy/ethpy/hyperdrive/api.py

```diff
@@ -259,6 +259,93 @@ def get_in_for_out(
         )
         return FixedPoint(scaled_value=int(in_for_out))
 
+    def calculate_fees_out_given_bonds_in(
+        self, bonds_in: FixedPoint, maturity_time: int | None = None
+    ) -> tuple[FixedPoint, FixedPoint, FixedPoint]:
+        """Calculates the fees that go to the LPs and governance.
+
+        Implements the formula:
+            curve_fee = ((1 - p) * phi_curve * d_y * t)/c
+            gov_fee = curve_fee * phi_gov
+            flat_fee = (d_y * (1 - t) * phi_flat) / c
+
+        Arguments
+        ---------
+        bonds_in : FixedPoint
+            The amount of bonds in.
+        interface : HyperdriveInterface
+            The API interface object.
+        maturity_time : int, optional
+            The maturity timestamp of the open position, in epoch seconds.
+
+        Returns
+        -------
+        tuple[FixedPoint, FixedPoint, FixedPoint] consisting of:
+            curve_fee : FixedPoint
+                Curve fee, in shares.
+            flat_fee : FixedPoint
+                Flat fee, in shares.
+            gov_fee : FixedPoint
+                Governance fee, in shares.
+        """
+        if maturity_time is None:
+            maturity_time: int = self.pool_config["positionDuration"]
```

Comment:
I'm not sure about this line, but you have more context.

Reply:
should be `self.current_block_time + self.pool_config["positionDuration"]`
so that after `time_remaining_in_seconds = maturity_time - self.current_block_time` resolves to `self.pool_config["positionDuration"]`
and after `normalized_time_remaining = time_remaining_in_seconds / self.pool_config["positionDuration"]` resolves to 1
can we skip these steps and just set `normalized_time_remaining` to 1? to avoid mistakes like these, and avoid mental math

Reply:
I'd rather keep the pattern of setting maturity time to `self.current_block_time + self.pool_config["positionDuration"]`